### DISCUSSION TITLE
New extensions (User Spooled Files and Hawkeye Pathfinder) based on codefori

### DISCUSSION THREAD

**Discussion by m-tyler** [codefori/vscode-ibmi#2160] [General]
I was able to get two side projects finished and ready for publishing.

## User spooled files
Market place link: https://marketplace.visualstudio.com/items?itemName=m-tyler.vscode-ibmi-UserSpooledFiles

## Hawkeye Pathfinder
Market place link: https://marketplace.visualstudio.com/items?itemName=m-tyler.vscode-ibmi-hawkeye

**Comment by worksofliam** [comment]
I would love to see some screenshots of the Hawkeye extension in action!

**Comment by m-tyler** [comment]
I added some quickly, not all tools but they all function very similar and return results to the same view.

**Comment by thomprl** [comment]
I get this error when trying to use the spool file browser.
`Cannot read properties of undefined (reading 'map')`

**Reply by m-tyler** [reply]
I don't see that error when I run the extension.  Please create an issue on the GitHub project page and let me know what version of Code for i product you have and what steps you were using before the error appeared.

Thanks, Matt

**Reply by thomprl** [reply]
I assume this IBM idea is yours?  https://ideas.ibm.com/ideas/IBMI-I-4161  I gave it an Upvote either way.
